src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6398,28 +6398,34 @@ def test_aks_create_with_web_application_routing(self, resource_group, resource_
 
     @AllowLargeResponse()
     @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
-    def test_aks_create_web_application_routing_dns_zone_not_exist(self, resource_group, resource_group_location):
+    def test_aks_create_web_application_routing_with_private_dns_zone(self, resource_group, resource_group_location):
```

Comment:
Let me check with you if the origina test name 'test_aks_create_web_application_routing_dns_zone_not_exist' already not exists. If so, please check if the recording file test_aks_create_web_application_routing_dns_zone_not_exist.yaml has been removed.